COPYRIGHT

```diff
@@ -1,7 +1,7 @@
 Quarto
-Copyright (C) 2020-2021 RStudio, PBC
+Copyright (C) 2020-2023 Posit Software, PBC
```

Comment:
2022, right?

Reply:
Yes. This was the only manual change it did. Will fix.